src/config/dynamic-theme-fixes.config

```diff
@@ -3421,6 +3421,13 @@ a[href*="about/products"]
 
 ================================
 
+google.com/search
```

Comment:
Hmmm, al existing Google fixes will be lost trough this, please add the INVERT to `www.google.*` around L8543

Reply:
Done in fd3881c